src/runtime/image.ts

```diff
@@ -161,8 +161,10 @@ function getPreset (ctx: ImageCTX, name?: string): ImageOptions {
 function getSizes (ctx: ImageCTX, input: string, opts: ImageSizesOptions) {
   const width = parseSize(opts.modifiers?.width)
   const height = parseSize(opts.modifiers?.height)
+  const densities = opts.densities ? parseDensities(opts.densities) : [1, 2]
```

Comment:
this would mean a default of `1x 2x` ?

Not sure if 2x should be added by default. This could lead to confusion for many users. 🤔 
Further, it would change the behaviour on upgrade. 

Suggested improvement: allow to configure the default globally via _nuxt.config_.